sheetify/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for sheetify 6.0
+// Project: https://github.com/stackcss/sheetify#readme
+// Definitions by: Todd Kennedy <https://github.com/toddself>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace sheetify {
+  export function getPrefix(css: string): string;
+}
+
+declare function sheetify(src: string | TemplateStringsArray, filename?: string, options?: {[prop: string]: any}, done?: (err: Error, css?: string, prefix?: string) => void): string;
```

Comment:
See "Optional Parameters in Callbacks" [here](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).